moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -1,5 +1,5 @@
 export const filterProjectTeamMembers = value => {
-  if (value === " :") {
+  if (!value || value === " :") {
```

Comment:
Can we get rid of the `value === " :"` check here since you solved it with your query above?